webui/ocw/lib/vault.py

```diff
@@ -84,8 +73,8 @@ class AzureCredential(Vault):
     ''' Known data fields: subscription_id, client_id, client_secret, tenant_id
     '''
 
-    def __init__(self, url=None, user=None, password=None, certificate_dir=None):
-        return super().__init__(url, user, password, certificate_dir)
+    def __init__(self):
```

Comment:
As long as we do not need to do something in that initializer, we don't need to have it at all! And then the super() initializer is called anyway.

Reply:
I was not sure how constructors works in python so decided to leave it . Now I will remove it